src/test/js/multibranch/noRunsActivityPage.js

```diff
@@ -0,0 +1,24 @@
+const jobName = 'noRunsActivityPage';
+/** @module failing
+ * @memberof testcases
+ * @description TEST: basic tests around the failing pipeline. Test whether the result is not collapsed.
+ */
+module.exports = {
+    /**
+     * Create Multibranch Pipeline Job "noRunsActivityPage" with no branches.
+     * @param browser
+     */
+    'Step 01': function (browser) {
+        var multibranchCreate = browser.page.multibranchCreate().navigate();
+      
+        multibranchCreate.createBranch(jobName);
+    },
+    /**
+     * Make sure that we show the empty state.
+     */
+    'Step 02': function (browser) {
+         var blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName, 'jenkins');
+
+         blueActivityPage.waitForElementVisible('@emptyStateShoes');
+    }
+}
```

Comment:
@imeredith Looks good Ivan.

So one way you could advance this a bit further is for us to add a reusable command/assertion function doing more detailed checking on the Empty State (more than just checking that it exists). [bluePipelineActivity#assertBasicLayoutOkay](https://github.com/jenkinsci/blueocean-acceptance-test/blob/master/src/main/js/page_objects/blueocean/bluePipelineActivity.js#L42-L58) would be an example of that kind of function.